pages/projects/[owner]/[repo].tsx

```diff
@@ -15,7 +15,11 @@ function project(props: githubFullinfo) {
     projectname: props.projectname,
   };
 
-  return <Layout descriptionText={ProjectPage(projectFullInfo)}></Layout>;
+  return (
+    <Layout descriptionText={ProjectPage(projectFullInfo)}>
+      <></>
```

Comment:
we fix the layout and now we have to give it a child, so for now until we will add the components i added <></>

Reply:
Please add a comment that explains it in the code.

Reply:
Added